### DISCUSSION TITLE
Frontend Developer? Comment here!

### DISCUSSION THREAD

**Discussion by webcrumbs-community** [webcrumbs-community/webcrumbs#115] [General]
Welcome, WebCrumbs frontend maestros!

Joining the conversation is a breeze:

- Drop a comment right here, or
- Get a shoutout from a fellow developer.

Stay tuned to this space. We'll be sharing the latest topics and challenges that spark your curiosity and matter most to you!

**Comment by webcrumbs-community** [comment]
I'll start by giving a shoutout to @jsimples, @Ashok-Kumar-dharanikota, @aditya4196, @rovoskarl, @AJBrownson, @Dinushan-S. Do you know of others that are frontend developers and may be interested in this thred?

**Comment by ShatilKhan** [comment]
Dropping in to work on some frontend issues

**Comment by dew-dr0p** [comment]
Happy to dive in as a frontend developer.

**Comment by webcrumbs-community** [comment]
New issues coming soon! 👀

**Comment by webcrumbs-community** [comment]
Added a new frontend issue at #156!

**Reply by webcrumbs-community** [reply]
@Ashok-Kumar-dharanikota, @aditya4196, @AJBrownson, @Dinushan-S, @ShatilKhan, @dew-dr0p

**Reply by ShatilKhan** [reply]
I've commented,please assign it to me

**Comment by webcrumbs-community** [comment]
Added **Task /admin: Design Plugin Management Page for Admin Panel on Figma** #157 !

**Comment by webcrumbs-community** [comment]
Hey, all! Dropped a new **Task /registry: Design Plugin Search Page for the Registry Website on Figma** #159

**Comment by shreykoradia** [comment]
Hey Hi I am shrey , I was a core contributor to web crumbs when it was in its initial stage , for a meanwhile I wasn't able to keep a track for it so would be happy again to contribute towards it so please help me if there is any issues or advancements that could be done and also yeah please any links where i could see the advancements done !

**Reply by jsimples** [reply]
Hey, Shrey! Good to see you.  

Best way to see where we are I think is by the main fronts:

`/admin`: The admin-panel frontend. [See open issues](https://github.com/webcrumbs-community/webcrumbs/labels/%2Fadmin)

`/registry`: Frontend of a website similar to npm or WordPress plugins that will host all plugins. [See open issues](https://github.com/webcrumbs-community/webcrumbs/labels/%2Fregistry)

`/core`: Where we're experimenting with ways to embed plugins (remotely, dynamically-loaded, ssr-rendered, client-side-hydrated...). [See open issues](https://github.com/webcrumbs-community/webcrumbs/labels/%2Fcore)

**Comment by webcrumbs-community** [comment]
This is important! For us to keep in mind in
 https://youtu.be/o8jEN-qg42k?si=RX9Mf-hc5TbrHRAR

**Comment by vinayBhoure** [comment]
Hi, I came here in search of frontend issues, assign me one.

**Comment by dew-dr0p** [comment]
Great work so far @webcrumbs-community.
I just noticed a lot of design tasks have been created and assigned. I can't wait to see the designs approved by the team. I would love to jump in to see the functionalities implemented.
As soon as new issues are opened to implement the functionalities of the designs created don't hesitate to mention me as I'd be happy to jump in and see that they're all implemented to pixel perfection.

**Reply by webcrumbs-community** [reply]
Thank you so much, @vinayBhoure and @dew-dr0p. I'll add more issues later today and let you know!

**Reply by webcrumbs-community** [reply]
@dew-dr0p and @vinayBhoure Please check open frontend issues here!
[Open frontend issues](https://github.com/webcrumbs-community/webcrumbs/issues?q=is%3Aopen+is%3Aissue+label%3A%40frontend)
This link will be always updated

**Reply by vinayBhoure** [reply]
Thanks, @webcrumbs-community for the updates regarding issues
